Contest_API.md

```diff
@@ -929,16 +933,16 @@ The following endpoints are associated with languages:
-| icpc\_id    | string         | no        | yes       | CDS        | external identifier from ICPC CMS                                                            |
-| team\_id    | ID             | yes       | no        | CDS        | [ team](#teams) of this team member                                               |
-| first\_name | string         | yes       | no        | CDS        | first name of team member                                                                    |
-| last\_name  | string         | yes       | no        | CDS        | last name of team member                                                                     |
-| sex         | string         | no        | yes       | CDS        | either `male` or `female`, or possibly `null`                                            |
-| role        | string         | yes       | no        | CDS        | one of `contestant` or `coach`                                                           |
-| photo       | array of IMAGE | no        | yes       | CDS        | registration photo of the team member. Only allowed mime types are image/jpeg and image/png. |
+| Name        | Type           | Required? | Nullable? | Source @WF | Description                                                                                                 |
+| ----------- | -------------- | --------- | --------- | ---------- | ----------------------------------------------------------------------------------------------------------- |
+| id          | ID             | yes       | no        | CDS        | identifier of the team-member                                                                               |
+| icpc\_id    | string         | no        | yes       | CDS        | external identifier from ICPC CMS                                                                           |
+| team\_id    | ID             | yes       | no        | CDS        | [ team](#teams) of this team member                                                                         |
+| first\_name | string         | yes       | no        | CDS        | first name of team member                                                                                   |
+| last\_name  | string         | yes       | no        | CDS        | last name of team member                                                                                    |
+| sex         | string         | no        | yes       | CDS        | either `male` or `female`, or possibly `null`                                                               |
+| role        | string         | yes       | no        | CDS        | one of `contestant` or `coach`                                                                              |
+| photo       | array of IMAGE | no        | yes       | CDS        | registration photo of the team member. Only allowed mime types are image/jpeg, image/png and image/svg+xml. |
```

Comment:
I’m fine with allowing all three formats everywhere

Reply:
> > Does a team photo in SVG format make sense?
> 
> No.

I didn't mean that to be a rhetorical question BTW: it could be that in some contests, people would like to use a logo for a team.  ;-)

Reply:
Also what does it hurt us to say images can be any of the three formats? I don't really see any issues here.
If you agree I will update all image fields to have all three formats.

Reply:
> Also what does it hurt us to say images can be any of the three formats? 

The "hurt" is that clients now have to support all those formats. OTOH, at least all those formats are web safe.

> I don't really see any issues here. If you agree I will update all image fields to have all three formats.

I guess that's fine?

Reply:
Ok I will change it. Clients already need SVG for countries so I don’t see a big harm. And indeed they are web safe. Nowadays you can even use img tags for svgs…

Reply:
I'm 👍 for the PR as is, but if we start adding all the reasonable image formats anyway, maybe we should just remove the restriction completely. I.e. remove the sentence "Only allowed mime types are image/jpeg, image/png and image/svg+xml." everywhere.

Reply:
I’m fine either way. But if we drop image format restrictions in the tables we should have it at the top once I believe. Otherwise you also need to support bmp, webp, heic, …
Maybe just leave it as is?

Reply:
I think it's fine as-is, but would +1 a change to make it consistent and have a 'supported image types' at the top that lists jpg, png, and svg for everything. Once you need to support something like svg for flags it's generally not hard to do it elsewhere, and might even be common code.

I'd be fine discussing adding other formats (later), but I wouldn't support making this open-ended. Each client language has different limitations and might need an additional library or workaround to support a new file type. We've got a good set with these and I don't want to make it harder for clients to adopt without real benefit or requests.

Reply:
I changed it such that the list of the three supported image mime types is listed at the top and nowhere else. I see no harm in doing this.